tests/security/pam/pam_su.pm

```diff
@@ -43,8 +44,12 @@ sub run {
     upload_logs($su_file);
     upload_logs($sul_file);
 
-    # Switch to user console
     clear_console;
+
+    # Make sure that non-root user has permissions for $serialdev to get openQA work properly
+    assert_script_run("chmod 666 /dev/$serialdev") if (is_s390x);
```

Comment:
Better put this command into some prepare modules such as consoletest_setup, system_prepare etc..
If this PR ONLY for fix security, i am ok for that.
This also failed in migration test cases.

Reply:
check utils::ensure_serialdev_permissions

Reply:
Thanks, https://bugzilla.suse.com/show_bug.cgi?id=1195620 was filed to track the issue as well.
Any yes, you are correct, let me revise my code once get update from the bug.

Reply:
@foursixnine @coolgw 
Due to file permission set to 1154, utils::ensure_serialdev_permissions seems not work fine. I re-start the test on s390x platform, and failed again.

http://openqa.suse.de/tests/8124207#

Let me enhance my code then